Reply/app/src/main/res/values/attrs.xml

```diff
@@ -24,6 +24,10 @@
     <attr name="emphasisMediumAlpha" format="float"/>
     <attr name="emphasisDisabledAlpha" format="float"/>
 
+    <attr name="motionInterpolatorPersistent" format="reference"/>
```

Comment:
thoughts on `motion` vs `transition` naming? i think `transition` is a little more clear and specific, but i wonder if we'd want to use these interpolators for animations that aren't technically transitions.

Reply:
I think this would apply to motion that wouldn't be considered a `transition`. Thinking about this made me realize I had forgotten to update EmailSwipeActionDrawable's interpolator:
https://github.com/material-components/material-components-android-examples/pull/8/files#diff-0ae3075bb45657852ba051b458714a03R82

Design video of above animation: 
https://storage.googleapis.com/spec-host-backup/mio-design%2Fassets%2F1SapnI_EXJwjkvMb87hWWakpTMLIKoxiQ%2F03-reply-inboxzero.mp4

This is an example of where I think the `transition` naming restricts where these attrs are expected to be applied. Thoughts?